db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/FailureLogger.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (C) Gustav Karlsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.kagkarlsson.scheduler;
+
+import com.github.kagkarlsson.scheduler.logging.LogLevel;
+import org.slf4j.Logger;
+
+class FailureLogger {
```

Comment:
I'd like to keep this class in the `logging` package, but at the same time I wanted it to be package-private as this is (arguably) not something we want to expose to the users... Or we could move the `LogLevel` to the main package and remove the `logging` package altogether. WDYT? 🤔

Reply:
It doesn't really bother me if it would be public, I haven't really been strict about that in other places. Also, it turned out quite generic, so `.logging` sounds good

Reply:
It became something more like `ConfigurableLogger` rather than something having anything to do with Failure :)

Reply:
`ConfigurableLogger` it is, then.